sdks/java/core/src/main/java/org/apache/beam/sdk/values/WindowingStrategy.java

```diff
@@ -47,7 +47,8 @@
   /** The accumulation modes that can be used with windowing. */
   public enum AccumulationMode {
     DISCARDING_FIRED_PANES,
-    ACCUMULATING_FIRED_PANES
+    ACCUMULATING_FIRED_PANES,
+    RETRACTING_FIRED_PANES,
```

Comment:
Since this is part of the public API surface, it would be best if we explicitly said that this was experimental.

Reply:
Thanks @lukecwik. I am not sure if I followed the best practice, but I add `@Experimental` annotation, add a comment and also add to javadoc about experimental information.

Reply:
It's actually a really nice catch that reminds me all future user facing change regarding to retractions should be marked as experimental plus necessary javadoc/comment warning.